src/Compilers/CSharp/Portable/Lowering/AsyncRewriter/AsyncRewriter.cs

```diff
@@ -173,7 +173,7 @@ protected override BoundStatement GenerateStateMachineCreation(LocalSymbol state
                 bodyBuilder.Add(
                     F.Assignment(
                         F.Field(F.Local(stateMachineVariable), _builderField.AsMember(frameType)),
-                        F.StaticCall(methodScopeAsyncMethodBuilderMemberCollection.BuilderType, "Create", ImmutableArray<TypeSymbol>.Empty)));
+                        F.StaticCall(BinderFlags.IgnoreAccessibility, methodScopeAsyncMethodBuilderMemberCollection.BuilderType, "Create", ImmutableArray<TypeSymbol>.Empty))); // TODO (acasey): conditional
```

Comment:
I'm not sure when we'd need a value other than BinderFlags.IgnoreAccessibility here. I've removed the comment (and added an async test). @amcasey: Please create a GitHub issue if necessary.